packages/mongodb-rag-core/src/MongoDbPageStore.ts

```diff
@@ -14,24 +14,37 @@ export type MongoDbPageStore = DatabaseConnection &
     loadPages(
       args?: LoadPagesArgs<Filter<PersistedPage>>
     ): Promise<PersistedPage[]>;
+    metadata: {
+      databaseName: string;
+      collectionName: string;
+    };
   };
 
+export type MakeMongoDbPageStoreParams = MakeMongoDbDatabaseConnectionParams & {
+  collectionName?: string;
```

Comment:
can you add a tsdoc comment about how default is "pages"

smthn like:

```suggestion
  /**
    @default "pages"
   */
  collectionName?: string;
```